t/cobrand/national_assets.t

```diff
@@ -0,0 +1,41 @@
+use FixMyStreet::TestMech;
+my $mech = FixMyStreet::TestMech->new;
+
+# Create test data
+my $body = $mech->create_body_ok( 2561, 'Bristol County Council' );
+
+
+subtest 'cobrand assets includes cobrand assets javascript', sub {
+    FixMyStreet::override_config {
+        ALLOWED_COBRANDS => [ 'fixmystreet' ],
+        MAPIT_URL => 'http://mapit.uk/',
+        COBRAND_ASSETS => [ 'buckinghamshire' ]
+    }, sub {
+        $mech->get_ok("/report/new?latitude=51.494885&longitude=-2.602237");
+        $mech->content_contains('buckinghamshire/assets.js');
+    };
+};
+
+subtest 'no assets js if cobrand not in cobrand_assets', sub {
+    FixMyStreet::override_config {
+        ALLOWED_COBRANDS => [ 'fixmystreet' ],
+        MAPIT_URL => 'http://mapit.uk/',
+        COBRAND_ASSETS => [ 'bromley' ]
+    }, sub {
+        $mech->get_ok("/report/new?latitude=51.494885&longitude=-2.602237");
+        $mech->content_lacks('buckinghamshire/assets.js');
+    };
+};
+
+subtest 'cobrand assets includes not applied on cobrand sites', sub {
```

Comment:
Should there be a test that ensures that a cobrand's `assets.js` is included in the cobrand even if `COBRAND_ASSETS` is/isn't set?